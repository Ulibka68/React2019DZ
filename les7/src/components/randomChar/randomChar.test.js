import React from "react";
import RandomChar from "./randomChar";
import renderer from "react-test-renderer";


describe('Test <RandomChar>', () => {
    it('Random have rendered correctly', () => {
        const char  =renderer.create(<RandomChar showRandom />).toJSON();
        expect(char).toMatchSnapshot();
    })
}) 
