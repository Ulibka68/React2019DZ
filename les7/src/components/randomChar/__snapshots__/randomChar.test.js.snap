// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <RandomChar> Random have rendered correctly 1`] = `
<div
  className="random-block rounded"
>
  <div
    className="lds-css ng-scope"
  >
    <div
      className="lds-spin"
    >
      <div>
        <div>
           
        </div>
      </div>
      <div>
        <div>
           
        </div>
      </div>
      <div>
        <div>
           
        </div>
      </div>
      <div>
        <div>
            
        </div>
      </div>
      <div>
        <div>
           
        </div>
      </div>
      <div>
        <div>
           
        </div>
      </div>
      <div>
        <div>
           
        </div>
      </div>
      <div>
        <div>
            
        </div>
      </div>
    </div>
  </div>
</div>
`;
